site/content/index.md

```diff
@@ -119,16 +119,56 @@ layout: unstyled
   <div className="prose dark:prose-invert mx-auto max-w-2xl px-4 lg:max-w-6xl lg:px-8 xl:px-12">
     <h2 className="text-center">Features</h2>
 
-    Here are some of the cool Flowershow features:
 
-    - CommonMark, GitHub Flavoured Markdown and Obsidian extensions support (e.g. wikilinks) [[syntax|➡ learn more]]
-    - MDX support for custom React components additions [[custom-components|➡ learn more]]
-    - Customizable layouts of your pages [[layouts|➡ learn more]]
-    - Tailwind support for easy styling in-markdown HTML and custom components [[tailwind|➡ learn more]]
-    - Google analytics support [[analytics|➡ learn more]]
-    - Basic site configurations using single JSON file [[basic-config|➡ learn more]]
+Here are some of the cool features that are currently supported by Flowershow, and some that we're intensively working on.
 
-    👷 There is a lot of stuff we're still working on.
+### ✅/🚧 Markdown
```

Comment:
I'd aim to have either ✅ or 🚧 rather than both. Either that means splitting sections or going for one or other as appropriate.

On this one i'd say it was ✅ - ok there may be a few things but that isn't the end of the world.